lib/gulp-tasks/file-globbing.js

```diff
@@ -0,0 +1,130 @@
+/**
+ *  @fileOverview Uses Gulpjs and gulp-inject to glob files
+ *
+ *  @author       Scott Nath
+ *
+ *  @requires     NPM:gulp-inject
+ *  @requires     ./lib/get-options
+ */
+'use strict';
+
+var inject = require('gulp-inject'),
+  path = require('path'),
+  getOptions = require('../get-options').getOptions;
+
+/**
+ * Function to get default options for an implementation of gulp-inject
+ * @function
+ * @name getDefaultOptions
+ * 
+ * @returns {Object}  options  an object of options
+ */
+function getDefaultOptions () {
+
+  var options = {
+    config: {
```

Comment:
could you document options?
It's not clear why src is a file and dest is not, what is dependencies responsible for, what is config.relative used for and etc